dot/utils_integration_test.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+//go:build integration
+// +build integration
+
+package dot
+
+import (
+	"log"
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/ChainSafe/gossamer/lib/trie"
+	"github.com/stretchr/testify/require"
+)
+
+// TestNewConfig tests the NewTestConfig method
+func TestNewConfig(t *testing.T) {
+	cfg := NewTestConfig(t)
+	require.NotNil(t, cfg)
+}
+
+// TestNewConfigAndFile tests the NewTestConfigWithFile method
+func TestNewConfigAndFile(t *testing.T) {
+	testCfg, testCfgFile := newTestConfigWithFile(t)
+	require.NotNil(t, testCfg)
+	require.NotNil(t, testCfgFile)
+}
+
+// TestInitNode
+func TestNewTestGenesis(t *testing.T) {
+	cfg := NewTestConfig(t)
+	require.NotNil(t, cfg)
```

Comment:
Perhaps batch remove all `require.NotNil(t, cfg)`? It looks like we're testing test code which doesn't make much sense.

Reply:
Good point, I've removed these.